WebView/app/src/main/res/layout/activity_main.xml

```diff
@@ -19,7 +19,8 @@
     android:layout_width="fill_parent"
     android:layout_height="fill_parent"
     xmlns:tools="http://schemas.android.com/tools"
-    tools:context=".MainActivity">
+    tools:context=".MainActivity"
+    android:configChanges="uiMode">
```

Comment:
What is this?

Reply:
That was meant to be on a different branch and I forgot to remove it from this one, I will take it out